apps/web/src/server/applications/common/isCaseWelsh.js

```diff
@@ -12,3 +13,20 @@ export const isCaseRegionWales = (regions) => {
 
 	return Boolean(regions.find((region) => region.name === 'wales'));
 };
+
+/**
+ * @param {number} caseId
+ * @returns {Promise<boolean>}
+ * */
+export const isCaseWelsh = async (caseId) => {
```

Comment:
feel free to say no to this one Hadrian . . .
initially I was confused with the isCaseRegionWales and isCaseWelsh fn names.  I am wondering whether it is clear to change the isCaseWelsh name to something more reflective of "do we display the welsh fields"?  eg isCaseDisplayedWithWelshFields.  hmm, what are your thoughts?

Reply:
This function is general purpose rather than specific to the use case of checking whether to display the Welsh fields. It just check the regions to determine if one of them is Welsh, similar to the `isCaseRegionWales` function but you only need to pass it the case ID and it performs a fetch for you.

I could make the name for descriptive though, something like `isCaseInWales`. What do you think?

Reply:
hmm, that still sounds like the same name as isCaseRegionWales to me.  Maybe this is me overthinking it - No worries, proceed without changes.  I will approve.